src/main/java/net/minecraftforge/common/loot/IGlobalLootModifier.java

```diff
@@ -0,0 +1,10 @@
+package net.minecraftforge.common.loot;
+
+import java.util.List;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.world.storage.loot.LootContext;
+
+public interface IGlobalLootModifier {
+    List<ItemStack> apply(List<ItemStack> generatedLoot, LootContext context);
```

Comment:
This seems to be the core of the system, yet ZERO docs?

Reply:
I'll address this. I didn't even want to open a PR originally because I was bashing around blindly, but I got zero feedback.